Microsoft.DotNet.Interactive.Formatting/Formatter{T}.cs

```diff
@@ -71,6 +73,7 @@ void Initialize()
         /// </summary>
         /// <param name="obj">The obj.</param>
         /// <param name="writer">The writer.</param>
+        /// <param name="mimeType">The mimeType</param>
         public static void FormatTo(
```

Comment:
Let's not add XML documentation until and unless it's meaningful.

Reply:
done